### PR TITLE
Progress update/Image editor update

### PR DESCRIPTION
**Pull request by jaydenanderson**

### CODE REVIEW

File: blocks/el/radial-progress/index.ess [hunk 1]
```diff
@@ -21,12 +21,53 @@ var array = new Array(101)
...
+        animation pulselight 2s linear 0s infinite
+        border = '2px solid ' + __.color.primary
+        border-radius 10000px
+        content ''
+        display block
+        left 50%
+        position absolute
+        top 50%
+        transform translate(-50%,-50%)
+        animation pulseGrow 1s linear 0s infinite
```
Comment (mndvns) [inline blocks/el/radial-progress/index.ess]: won't this just override the other `animation`?

Comment (jaydenanderson) [inline blocks/el/radial-progress/index.ess]: Nah. The other animation is a transition.

Comment (jaydenanderson) [inline blocks/el/radial-progress/index.ess]: Oh derp.

Comment (jaydenanderson) [inline blocks/el/radial-progress/index.ess]: I derped.

Comment (jaydenanderson) [inline blocks/el/radial-progress/index.ess]: SOMETIMES YOU NEED TO INCLUDE ANIMATION TWICE BECAUSE OTHERWISE THE TERRORISTS WILL WIN.

**Comment by jakerichan** [comment]
:+1:

**Pull request merged**